xbmc/platform/darwin/ios/XBMCApplication.mm

```diff
@@ -191,11 +185,11 @@ static void XBMCsendEvent(id _self, SEL _cmd, UIEvent *event)
       if (eventType == GSEVENT_TYPE_KEYUP)
       {
         // support 32 and 64bit arm here...
-        int idx = GSEVENTKEY_KEYCODE;
+        int idx;
```

Comment:
assign either `GSEVENTKEY_KEYCODE_32_BIT` or `GSEVENTKEY_KEYCODE_64_BIT` and remove one of the if/else assignments.

Reply:
I think it's even possible to use conditional compilation here with `#if TARGET_RT_64_BIT`

> TARGET_RT_64_BIT        - Generated code uses 64-bit pointers